casper/src/test/scala/coop/rchain/casper/util/rholang/InterpreterUtilTest.scala

```diff
@@ -58,24 +58,25 @@ class InterpreterUtilTest
     )
 
   "computeBlockCheckpoint" should "compute the final post-state of a chain properly" in effectTest {
+    val time = System.currentTimeMillis()
```

Comment:
Why not use LogicalTime?

Reply:
It's in the `comm` layer

Reply:
I'm going to replace `currentTimeMillis` with `0`